docs/api.rst

```diff
@@ -83,6 +83,7 @@ developers, not a gospel.
     api/pulp_smash.tests.rpm.cli
     api/pulp_smash.tests.rpm.cli.test_copy_units
     api/pulp_smash.tests.rpm.cli.test_search
+    api/pulp_smash.tests.rpm.cli.test_upload_and_remove_langpacks
```

Comment:
I am wondering if we should call the  module `test_langpacks` and the inside the module have the `UploadAndRemoveTestCase`. I am thinking about this because we may want to do more operation with langpacks and having all that inside the same module sounds better to me.